js/main.js

```diff
@@ -45,15 +60,25 @@ const populateList = (data) => {
     `;
   }
 
+  // Set snippet of HTML for publisher countries
+  if (data.publisher.country) {
+    countryHTML = `
+      <p class="country">
+        (${data.publisher.country && countries.getName(data.publisher.country.toUpperCase(), 'en')})
+      </a>
```

Comment:
Should this be `</p>`?
```suggestion
      </p>
```

Reply:
Whoops, yes it should be. Fixed!